src/Symfony/Component/String/Tests/AbstractUnicodeTestCase.php

```diff
@@ -468,10 +468,10 @@ public static function provideBeforeAfterLastIgnoreCase(): array
         );
     }
 
-    public static function provideToFoldedCase(): array
+    public static function provideFolded(): array
```

Comment:
We're missing a place where this is consumed, isn't it?

Reply:
It's called by the parent test (AbstractAsciiTestCase)

```php

    /**
     * @dataProvider provideFolded
     */
    public function testFolded(string $expected, string $origin)
    {
        $this->assertEquals(
            static::createFromString($expected),
            static::createFromString($origin)->folded()
        );
    }

    public static function provideFolded()
    {
        return [
            ['hello', 'HELlo'],
            ['world', 'worLd'],
        ];
    }
```